src/devices/video/k051316.cpp

```diff
@@ -186,7 +196,7 @@ void k051316_device::device_start()
 
 void k051316_device::device_reset()
 {
-	memset(m_ctrlram, 0, 0x10);
+	std::fill(std::begin(m_ctrlram), std::end(m_ctrlram), 0);
 }
```

Comment:
I just realised this causes a change in behaviour.  Previously, readout enable and flip bits were part of this array, so they would be cleared on device reset.  With them no longer in the array, they’re unaffected by reset.  Should they be set to initial values on reset, or only on start?

Reply:
The chip has no reset line, for meh :-)

Reply:
Indeed, strangely enough. Because of that all games seem to configure all registers at startup / after a reset.